pkg/cloud/powervs.go

```diff
@@ -60,7 +61,16 @@ func NewPowerVSCloud(cloudInstanceID, zone string, debug bool) (Cloud, error) {
 }
 
 func newPowerVSCloud(cloudInstanceID, zone string, debug bool) (Cloud, error) {
-	apikey := os.Getenv("IBMCLOUD_API_KEY")
```

Comment:
I see you are completely removing the reading the API key from the env which is a breaking change, please keep both the ways.

Reply:
Done